tron/commands/display.py

```diff
@@ -352,6 +352,8 @@ class DisplayActionRuns(TableDisplay):
         ('End time', 'end_time'),
         ('Final exit status', 'exit_status'),
         ('Retry exit statuses', 'exit_statuses'),
+        ('Waits for triggers', 'triggered_by'),
```

Comment:
this shows all of the triggers it waits for, right? is there a way to show the triggers it has received?

also, might be worth renaming as `Blocked on` and `Triggers Published:`

Reply:
it will list all triggers and `(done)` next to those that were already published